src/EditorFeatures/CSharpTest/Diagnostics/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -192,7 +192,7 @@ public void TestConflictingProperties()
         {
             Test(
 @"class Test : [|I1|] { int Prop { get ; set ; } } interface I1 { int Prop { get ; set ; } } ",
-@"using System ; class Test : I1 { int I1 . Prop { get { throw new NotImplementedException ( ) ; } set { throw new NotImplementedException ( ) ; } } int Prop { get ; set ; } } interface I1 { int Prop { get ; set ; } } ");
+@"class Test : I1 { int I1 . Prop { get ; set ; } int Prop { get ; set ; } } interface I1 { int Prop { get ; set ; } } ");
```

Comment:
@DustinCampbell ported the bug to github https://github.com/dotnet/roslyn/issues/5898.  The problem is that VB used to prefer to generate autoprops and we broke those people,  The minimal fix here would be to only generate autoprops for VB users and let C# folks delete all those property bodies.